integration_tests/integration_test.go

```diff
@@ -247,100 +312,32 @@ var _ = Describe("LiveIntegrationTests", func() {
 	})
 
 	Describe("bigquery", func() {
-
-		var (
-			bqProvisionDetails   models.ProvisionDetails
-			bqDeprovisionDetails models.DeprovisionDetails
-			service              *googlebigquery.Service
-			datasetName          string
-		)
-
-		BeforeEach(func() {
-			datasetName = "integration_test_dataset"
-
-			bqProvisionDetails = models.ProvisionDetails{
-				ServiceID:     serviceNameToId[brokers.BigqueryName],
-				PlanID:        someBigQueryPlanId,
-				RawParameters: []byte("{\"name\": \"integration_test_dataset\"}"),
-			}
-
-			bqDeprovisionDetails = models.DeprovisionDetails{
-				ServiceID: serviceNameToId[brokers.BigqueryName],
-				PlanID:    someBigQueryPlanId,
-			}
-
-			service, err = googlebigquery.New(gcpBroker.GCPClient)
-			if err != nil {
-				panic("error creating bigquery client for testing")
+		It("can provision/bind/unbind/deprovision", func() {
+			service, err := googlebigquery.New(gcpBroker.GCPClient)
+			Expect(err).NotTo(HaveOccurred())
+
+			datasetName := "integration_test_dataset"
+			params := &genericService{
+				serviceId:          serviceNameToId[brokers.BigqueryName],
+				planId:             serviceNameToPlanId[brokers.BigqueryName],
+				bindingId:          "integration_test_bind",
+				instanceId:         datasetName,
+				rawProvisionParams: []byte("{\"name\": \"" + datasetName + "\"}"),
+				rawBindingParams: map[string]interface{}{
+					"role": "editor",
```

Comment:
would like to see this as a curated role, since that's what we're recommending to users. bigquery.admin should work.